### PR TITLE
styling(CreateGithubSync): preview repository

### PR DESCRIPTION
**Pull request by anish-patro**
Close: #7131 

## PR Details

preview repository button was overlapping because of wrong className

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by morhag90 — CHANGES_REQUESTED** [review]
Hi @anish-patro, 
The right solution will be to delete this css class (delete the entire file ), then clean the component from all uses and add wrap it with the new component : FlexItem.
Let me know if you have any question. 
Thanks :)

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGithubSync.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { Icon, ToggleField, Tooltip } from "@amplication/ui/design-system";
 import AuthWithGitProvider from "../../git/AuthWithGitProvider";
 
 const TOOLTIP_DIRECTION = "n";
-const CLASS_NAME = "create-git-sync";
+const CLASS_NAME = "create-git-sync1";
```
Comment (barshimi) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGithubSync.tsx:17]: why not using the same class ?

Comment (anish-patro) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGithubSync.tsx:17]: > why not using the same class ?

Previously it was not same, i changed that.

**Comment by anish-patro** [comment]
> Hi @anish-patro, The right solution will be to delete this CSS class (delete the entire file ), then clean the component from all uses and wrap it with the new component: FlexItem. Feel free to let me know if you have any questions. Thanks :)

i didn't understand, do you want me to change the UI?

**Comment by anish-patro** [comment]
the current ui contains the flex box in which all items are aligned properly, so do you want me to change anything further? I have used css another component seeing another file structures.

**Comment by anish-patro** [comment]
> @anish-patro - we're sorry for the long delay with our response here. Indeed, you need to change the way the layout is performed in the `CreateGithubSync.tsx` file by using FlexItem for the items and their alignment. You can search for `FlexItem` usage in other `.tsx` files to see how it's done. Once you move to use `FlexItem` you will no longer need the `CreateGithubSync.scss` file and will be able to delete it completely.
> 
> Please indicate in a comment here if you want to take this task or not so I will know if you want to work on it and if I should leave the PR open or not.
> 
> If you have any questions about the required implementation, feel free to ask them here and @-tag me and @morhag90

Yes I will work on this issue.

**Comment by overbit** [comment]
@anish-patro as there wasn't been any progress in the last month, I'm closing this PR. Feel free to open a new PR once you have a fix for the issue :)

**Pull request closed without merging**